### ISSUE TITLE
com.android.providers.downloads.ui is an important view

### ISSUE THREAD

**Issue by MrHamidKhan** [jaredsburrows/Android-Bloatware#5]
com.android.providers.downloads.ui lists down the current downloads and manage the proper post-action strategy. Without a separate download-manager, a user would have issues if they delete it.

Source code:
http://www.javased.com/index.php?source_dir=platform_packages_providers_downloadprovider/ui/src/com/android/providers/downloads/ui/DownloadList.java

**Comment by ferrolho** [comment]
Yup, I was just bitten by this! Deleted that and suddenly a few things stopped working. E.g., restoring photos from Google Photos by saving them to the device (from the cloud) failed and crashed the app.
